kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportPage.vue

```diff
@@ -195,12 +174,21 @@
       quizLink(quizId) {
         return this.classRoute(PageNames.REPORTS_LEARNER_REPORT_QUIZ_PAGE_ROOT, { quizId });
       },
-    },
-    $trs: {
-      reportLearners: {
-        message: 'Report learners',
-        context: 'Labels the Reports > Learners tab for screen reader users',
+      loadMoreLessonTable(){
+        if(this.limit > 20){
```

Comment:
This seems to suggest that the button should be forcefully dismissed after two clicks of the 'View more..' button. Is this intended?

Reply:
Yes!
>For both Lessons and quizzes display the first 10 items of the array
>If there are >10 Items, display a view more button
>On click, render another 10 items
> If still more items, on click, view the rest, no matter how many)
> If there are no more items to be shown (the user is viewing all), hide the "view more" button

From my understanding , we will render the first 20 and and that happens after 2 clicks. we can then hide the button because we will be displaying the rest.